src/kibana/plugins/kibana/_timepicker.js

```diff
@@ -24,7 +24,9 @@ define(function (require) {
       // Close if already open
       if ($scope.globalConfigTemplate === timepickerHtml) {
         delete $scope.globalConfigTemplate;
+        document.getElementById('timepicker').setAttribute('aria-expanded', 'false');
```

Comment:
Eh, no other way to do this? Without going into the DOM from here?

Maybe a small directive similar to ngClass but for attributes?